### ISSUE TITLE
Save to Google Drive

### ISSUE THREAD

**Issue by diboma** [drichard/mindmaps#71]
When I want to open a mindmap from cloud storage, Google Drive is among the options.
Howerver, when I want to save a mindmap to the cloud, Google Drive is not in the list.

**Comment by drichard** [comment]
Yeah, I know. To be honest, I don't know why Google Drive doesn't appear. I suspect it's an issue with the filestack picker.

**Comment by darkskiez** [comment]
Maybe you need to refresh the oauth credentials in the filestack website:

https://www.filestack.com/docs/tutorials/google-drive-oauth/
